fastlane/Fastfile

```diff
@@ -93,4 +93,42 @@ platform :ios do
       changelog: commit[:message]
     )
   end
+  
+  desc "Publish a user study build to TestFlight"
+  lane :beta_user_study do
+    signin
+    increment_build_number(
+      {
+        build_number: latest_testflight_build_number + 1
+      }
+    )
+
+    update_info_plist(
+      plist_path: "LLMonFHIR/Supporting Files/Info.plist", 
+      block: proc do |plist|
+        plist["UserStudyEnabled"] = true
+      end
+    )
+    
+    update_info_plist(
+      plist_path: "LLMonFHIR/Supporting Files/OpenAIConfig.plist", 
+      block: proc do |plist|
+        plist["OpenAI_API_Key"] = ENV["GOOGLE_APPLICATION_CREDENTIALS_BASE64"]
```

Comment:
After [#103](https://github.com/StanfordBDHG/.github/pull/103/) is merged, we should pass `OPENAI_API_KEY` instead of `GOOGLE_APPLICATION_CREDENTIALS_BASE64`. The latter is passed just for the sake of testing this workflow.

cc: @PSchmiedmayer